source/routing/raptor.cpp

```diff
@@ -535,23 +540,17 @@ RAPTOR::isochrone(const map_stop_point_duration& departures,
                   const std::vector<std::string>& forbidden,
                   const std::vector<std::string>& allowed,
                   bool clockwise,
-                  const nt::RTLevel rt_level) {
-    const DateTime bound = limit_bound(clockwise, departure_datetime, b);
+                  const nt::RTLevel rt_level)
+{
     set_valid_jp_and_jpp(DateTimeUtils::date(departure_datetime),
                          accessibilite_params,
                          forbidden,
                          allowed,
                          rt_level);
-    assert(data.dataRaptor->cached_next_st_manager);
-    next_st = data.dataRaptor->cached_next_st_manager->load(
```

Comment:
so the next stoptimes cache is removed? am i missing something?

Reply:
You're missing that the function now calls `first_raptor_loop()` that do the caching ;-)